jscomp/ml/ctype.ml

```diff
@@ -3674,7 +3674,7 @@ let rec subtype_rec env trace t1 t2 cstrs =
             true (* handled in the fields checks *)
           | Record_unboxed b1, Record_unboxed b2 -> b1 = b2
           | Record_inlined _, Record_inlined _ -> repr1 = repr2
-          | Record_extension, Record_extension -> true
+          | Record_extension b1, Record_extension b2 -> b1.is_exception = b2.is_exception
```

Comment:
Is this casting any exception to any other exception?
OK seems legit.

Reply:
I actually don't know what the code is about, I just made it compile in the most safe way :)